src/main/java/tds/exam/results/configuration/messaging/ExamResultsTransmitterMessageRecoverer.java

```diff
@@ -0,0 +1,17 @@
+package tds.exam.results.configuration.messaging;
+
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.rabbit.retry.RejectAndDontRequeueRecoverer;
+
+/**
+ * Handles the message after it has been rejected the required number of times.
+ */
+public class ExamResultsTransmitterMessageRecoverer extends RejectAndDontRequeueRecoverer {
+    private static final org.slf4j.Logger log = LoggerFactory.getLogger(ExamResultsTransmitterMessageRecoverer.class);
+
+    @Override
+    public void recover(final Message message, final Throwable cause) {
```

Comment:
This is only called after the last attempt.  So if the configuration is set to 3 this would be given the message on after the third attempt.